.github/workflows/labeler.yml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
```

Comment:
:thought_balloon: Maybe take the `issue-add-support-label.yml` workflow and [extend it for PRs](https://github.com/github/issue-labeler?tab=readme-ov-file#pull-request-support) instead of adding a new one.

Reply:
I already checked [issue_labeler](https://github.com/github/issue-labeler), but it does not analyse the changed files in a pull request, it just checks the title and the body.